.github/workflows/smoke.yml

```diff
@@ -14,7 +14,7 @@ concurrency:
 
 env:
   GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  SMOKE_TEST_BRANCH: main
+  SMOKE_TEST_BRANCH: revert-229-revert-228-amazimbe/use-new-maven-version
```

Comment:
Please revert this before merging. It's ok to say "the smoke test failure is expected" when making changes like this. Smoke test failures are not required checks so it doesn't block a merge.